params/config.go

```diff
@@ -186,6 +186,7 @@ type GovernanceConfig struct {
 	GoverningNode  common.Address `json:"governingNode"`
 	GovernanceMode string         `json:"governanceMode"`
 	Reward         *RewardConfig  `json:"reward,omitempty"`
+	KIP71          *KIP71Config   `json:kip71,omitempty`
```

Comment:
Does It needs to double quote?
`json:"kip71,omitempty"`

Reply:
yes right, thanks

Reply:
I think it need to add test case that  encode / decode GovernanceConfig variable to json.

Reply:
the last commit changed the double quote.

Reply:
You looked at the 2 steps previous commit.

Reply:
I also check the json encode/decode test by (read/write) (from/to) DB for `GovernanceConfig` by `NewGovernanceInitialize( )`
```go
func NewGovernanceInitialize(chainConfig *params.ChainConfig, dbm database.DBManager) *Governance {
	ret := NewGovernance(chainConfig, dbm)
	// nil is for testing or simple function usage
	if dbm != nil {
		ret.ReadGovernanceState()
		if err := ret.initializeCache(); err != nil {
			// If this is the first time to run, store governance information for genesis block on database
			cfg := GetGovernanceItemsFromChainConfig(chainConfig)
			if err := ret.WriteGovernance(0, cfg, NewGovernanceSet()); err != nil {
				logger.Crit("Error in writing governance information", "err", err)
			}
			// If failed again after writing governance, stop booting up
			if err = ret.initializeCache(); err != nil {
				logger.Crit("No governance cache index found in a database", "err", err)
			}
		}
	}
	return ret
}
```